tests/integration/benchmark/llmperf/scripts/run_benchmark.py

```diff
@@ -0,0 +1,514 @@
+                        "Value": container,
+                    },
+                    {
+                        "Name": "InstanceType",
+                        "Value": instance_type,
+                    },
+                    {
+                        "Name": "Concurrency",
+                        "Value": concurrency,
+                    },
+                ],
+                "Unit":
+                metrics[field_name]["unit"],
+                "Value":
+                float(field_value),
+            }]
+            response = cw.put_metric_data(Namespace=metrics_namespace,
+                                          MetricData=metric_data)
+            logger.info(
+                "publish metric: %s, model: %s, endpoint: %s, container: %s, instance_type: %s, concurrency: %s, response: %s",
+                metrics[field_name]["metric_name"],
+                model_name,
+                endpoint_type,
+                container,
+                instance_type,
+                concurrency,
+                response,
+            )
+
+
+def is_valid_device(devices, ec2_instance_type):
+    for device in devices:
+        if device == ec2_instance_type:
+            return True
+    return False
+
+
+def run_bash_command(bash_command):
+    try:
+        logger.info(f"{bash_command}")
+        output = subprocess.run(bash_command,
+                                capture_output=True,
+                                text=True,
+                                check=True)
+
+        for line in output.stdout.splitlines():
+            logger.info(line)
+
+        for line in output.stderr.splitlines():
+            logger.error(line)
+
+        return output.stdout.strip()
+    except subprocess.CalledProcessError as e:
+        logger.error(f"Failed to run {bash_command}, error: {e}")
+        return None
+    except FileNotFoundError:
+        logger.error(f"Not found command: {bash_command}")
+        return None
+
+
+def pull_docker_image(container):
+    try:
+        image_name = get_image_name(container)
+        logger.info(f"image_name={image_name}")
+    except ValueError as e:
+        return None
+    bash_command = f"docker pull {image_name}"
+    if run_bash_command(bash_command.split()) is None:
+        return None
+    return image_name
+
+
+def build_parameter_str(parameter_list):
+    if len(parameter_list) == 0:
+        return ""
+    return " ".join(parameter_list)
+
+
+def launch_container(container, image, model, hf_token,
+                     container_log_file_name):
+    docker_parameters = build_parameter_str(
+        container.get("docker_parameters", []))
+    server_parameters = build_parameter_str(
+        container.get("server_parameters", []))
+
+    try:
+        container_name = container["container"]
+        docker_command = f"docker run --name {container_name} --rm -e HUGGING_FACE_HUB_TOKEN={hf_token} {docker_parameters} --init {image} {server_parameters}"
```

Comment:
Is HUGGING_FACE_HUB_TOKEN valid for all containers? HF docs show HF_TOKEN https://huggingface.co/docs/huggingface_hub/en/package_reference/environment_variables#hftoken.

Maybe the containers differ in how they expect this value to be passed, should we include HF_TOKEN as well?